src/js/components/Organization.js

```diff
@@ -113,6 +113,13 @@ export default class Organization extends Component {
 
   render () {
     const person = this.props.person;
+    let givenName = person.givenName;
+    // check to see if givenName includes single letter (middle initial)
+    // after first name, and trim it
+    if (givenName && givenName.indexOf(' ') === givenName.length - 2) {
```

Comment:
Ok, I've updated the code to use reg exp to remove middle initials (with and without a period). `givenName` doesn't include a last name (some LDAP entries just have a trailing middle initial after the first name).